Header header
int64 abs_position
int64 speed
